src/hotspot/share/compiler/compilerDirectives.hpp

```diff
@@ -34,11 +34,11 @@
 
   //      Directives flag name,    type, default value, compile command name
   #define compilerdirectives_common_flags(cflags) \
-    cflags(Enable,                  bool, false, X) \
-    cflags(Exclude,                 bool, false, X) \
-    cflags(BreakAtExecute,          bool, false, X) \
-    cflags(BreakAtCompile,          bool, false, X) \
-    cflags(Log,                     bool, LogCompilation, X) \
+    cflags(Enable,                  bool, false, Enable) \
+    cflags(Exclude,                 bool, false, Exclude) \
+    cflags(BreakAtExecute,          bool, false, BreakAtExecute) \
```

Comment:
The BreakAtFlags are missing defaults since CompileCommand uses the "break" option. If we are going to add them to the directives - we should go through all uses and make sure that this is the only flag is uses.

Reply:
break can't distinct breatAtCompile or breatAtExecution. I think it's good idea to separate them.
Debugging the parser or optimizations needs the first. The developers who debug codegen need the second one.  

Currently, directive->breakAtCompile and directive->breakAtExecution are in use.  Even BreakCommand is translated into them. 

There's a cleanup JDK-8255216. that's another task I will follow.